### DISCUSSION TITLE
Is the 10 pin ribbon cable pinout correct?

### DISCUSSION THREAD

**Discussion by darkroomai** [exit-failure/tds7000#4] [Q&A]
Hi,

Thank you very much for this info.  Can you please check the pinout on the 10 pin ribbon sown in picture labeled "female IDC connector on a IDC-serial adapter:"  It appears the top row, odd pins, is not labeled correctly.

Thank you,

Austin

**Comment by darkroomai** [comment]
I found this:

https://www.vogons.org/thumbs/31670_9d784d0a15ead5763f29f1dbf042a3b1/DTK3a.jpg

![DTK3a](https://github.com/user-attachments/assets/1be6d56f-e629-4414-9b99-b2c50164bbea)

**Comment by exit-failure** [comment]
Sorry for replying so late :-/

> It appears the top row, odd pins, is not labeled correctly.

Thanks for post the helpful infographic. The pinout I've given, indeed, is for a male connector. I've corrected that mistake.

To make matters worse and if I remember correctly, at some point I found a hint in a forum post, that there might be versions of the scope with a different pinout on the motherboard's connector. Unfortunately, I didn't find any further evidence, so I omitted that as to not spread any potentially false information.

**Comment by fujinobe** [comment]
Hi, 
thank you for your great description. As darkrommai already mentioned:
The connector labeled "female IDC connector on a IDC-serial adapter:" still shows the wrong pin count: 
1 2 3 5 9  instead of
1 3 5 7 9
